AUTHORS

```diff
@@ -0,0 +1,2 @@
+Original author: Z572@github.com (did 99.9% of work)
+Humble contributor: Grigory Shepelev | shegeley@github.com | (did the rest)
```

Comment:
email address should a real address, like me is 873216071@qq.com, and user can mailto me